wasm/README.md

```diff
@@ -1,9 +1,11 @@
-## Using Bergamot Translator in JavaScript
+# Using Bergamot Translator in JavaScript
 The example file `bergamot.html` in the folder `test_page` demonstrates how to use the bergamot translator in JavaScript via a `<script>` tag.
```

Comment:
Here is one minor concern: if the user reads `wasm-example` from sphinx docs, it is not clear where to find `test_page`. Maybe it is better change `test_page` to `wasm/test_page`.

Reply:
I have left a note in the beginning. Linking to this folder in the repository.